### ISSUE TITLE
xclip with fish_vi_key_bindings problem.

### ISSUE THREAD

**Issue by user-b** [fish-shell/fish-shell#6497]
Fish 3.0.2, xterm 349

Hello. When I try to yank/paste inside emacs with xclip on (xclip.el) I get this:

```
bind: Key with name “right” does not have any mapping
bind: Key with name “left” does not have any mapping
bind: Key with name “ppage” does not have any mapping
bind: Key with name “npage” does not have any mapping
bind: Key with name “btab” does not have any mapping
bind: Key with name “down” does not have any mapping
bind: Key with name “up” does not have any mapping
bind: Key with name “f1” does not have any mapping
bind: Key with name “right” does not have any mapping
bind: Key with name “left” does not have any mapping
bind: Key with name “ppage” does not have any mapping
bind: Key with name “npage” does not have any mapping
bind: Key with name “btab” does not have any mapping
bind: Key with name “down” does not have any mapping
bind: Key with name “up” does not have any mapping
bind: Key with name “f1” does not have any mapping
bind: Key with name “right” does not have any mapping
bind: Key with name “left” does not have any mapping
bind: Key with name “ppage” does not have any mapping
bind: Key with name “npage” does not have any mapping
bind: Key with name “btab” does not have any mapping
bind: Key with name “down” does not have any mapping
bind: Key with name “up” does not have any mapping
bind: Key with name “f1” does not have any mapping
<finally here is the content of the clipboard>

```

Xclip.el is a emacs script interfacing xclip, a shell command interfacing the X clipboard. If I do comment `fish_vi_key_bindings` in `~/.config/fish/config.fish`, it does works perfectly without having to restart emacs. It would be fine to wrap `fish_vi_key_bindings` with a conditional to detect if I'm starting fish from xterm or emacs (when yanking/pasting..), but I can't find how to do it.

[#4188](https://github.com/fish-shell/fish-shell/issues/4188) seems to be related.

**Comment by faho** [comment]
Fixed in master by silencing these errors

**Comment by faho** [comment]
Duplicate of #5853

**Comment by faho** [comment]
Okay, now that I'm home: We tried silencing that error for binding presets (not user's bindings, because presumably you'd want to know about this) in 3.0, but we missed it for the vi bindings. That was added in  3b532fc793bd66b8d2bd40165cd35a3bae733519 to fix 5853.

(and yes, github requires, as far as I can tell, a separate comment to mark an issue as a duplicate. There's seemingly no other way to set it)
